components/usage/pkg/db/workspace_instance.go

```diff
@@ -224,10 +223,8 @@ type WorkspaceInstanceForUsage struct {
 	Type               WorkspaceType  `gorm:"column:workspaceType;type:char;size:16;default:regular;" json:"workspaceType"`
 	UsageAttributionID AttributionID  `gorm:"column:usageAttributionId;type:varchar;size:60;" json:"usageAttributionId"`
 
-	CreationTime VarcharTime `gorm:"column:creationTime;type:varchar;size:255;" json:"creationTime"`
```

Comment:
Removes these fields as they are just confusing. They did not get removed originally when we switched to `stoppingTime`